js/index.js

```diff
@@ -44,3 +44,37 @@ messageForm.addEventListener("submit", function (event) {
   newMessage.appendChild(removeButton);
   messageList.appendChild(newMessage);
 });
+// AJAX
+// New XMLHttpRequest object
+let githubRequest = new XMLHttpRequest();
+// Open a request
+githubRequest.open("GET", "https://api.github.com/users/SokolovskayaE/repos");
+// Send the request
+githubRequest.send();
+
+// Add a "load" event listener
+githubRequest.addEventListener("load", (event) => {
+  if (githubRequest.status === 200) {
+    // Successful HTTP request
+    let repositories = JSON.parse(githubRequest.response); // Parse the response and store it in a variable named repositories
+    console.log(repositories);
+
+    // Select the #projects section by id and store it in a variable named projectSection
+    let projectSection = document.getElementById("projects");
+    // Query the projectSection (instead of the entire document) to find the <ul> element and store it in a variable named projectList
+    let projectList = projectSection.querySelector("ul");
+    // For loop to iterate over your repositories Array, starting at index 0
+    //for (let i = 0; i < repositories.length; i++) {
+    for (let repo of repositories) {
+      // Create a new list item (li) element and store it in a variable named project
+      let project = document.createElement("li");
+      // Set the inner text of your project variable to the current Array element's name property
+      //project.innerText = repositories[i].name;
+      project.innerText = repo.name;
+      // Transform your repository names into <a> tags that link to GitHub (hint: html_url property)
+      //project.innerHTML = `<a class="link link--no-decor" href="https://github.com//SokolovskayaE/fenrir-intro ">Intro to programming FENRIR</a>`;
```

Comment:
The lesson calls for creating an <a> element.  So like line 75, but instead of 'Intro to programming FENRIR', use repo.name.  Also, use repo.html_url for href=.  It is all inside `` so you can reference the variables using e.g. ${repo.name}.